Workbooks/WorkbooksMetadata.json

```diff
@@ -2295,5 +2295,18 @@
       "templateRelativePath": "LogAnalyticsQueryAnalysis.json",
       "subtitle": "",
       "provider": "Microsoft Sentinel Community"
-    } 
+    },
+    {
+      "workbookKey": "AsdEssential8",
+      "logoFileName": "",
+      "description": "This workbook provides insights on the health state of Azure resources against requirements by the ASD Essential 8.",
+      "dataTypesDependencies": [ "DeviceTvmSecureConfigurationAssessment" ],
+      "dataConnectorsDependencies": [],
+      "previewImagesFileNames": [ "AsdEssential8Black1.png", "AsdEssential8White1.png", "AsdEssential8Black2.png", "AsdEssential8White2.png" ],
+      "version": "1.0.0",
+      "title": "ASD Essential 8",
+      "templateRelativePath": "AsdEssential8.json",
+      "subtitle": "",
+      "provider": "Microsoft Corporation"
+    } 	
```

Comment:
Hey @nickt444, as it is a standalone content you need to add other parameters as well, such as "support", "author", "source" and "categories". please refer below image as reference. 
![image](https://user-images.githubusercontent.com/94349919/218446578-b76e9a0b-7dcb-4c43-802c-a8f6408e1c32.png)

Reply:
done